### PR TITLE
Update Windows implementation

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Hey @cco3,

Can you review these changes and make sure I didn't forget anything?

### CODE REVIEW

File: implementation-status.md [hunk 1]
```diff
@@ -29,15 +29,15 @@ permissions.query()       |           |         |     |       |         |
 permissions.revoke()      |           |         |     |       |         |
 watchAdvertisements()     |           |         |     |       |         |
 Persistent Device IDs     |           |         |     |       |         |
-GATT Server Connect       | ✓         | ✓       | ✓   | ✓     | [👷](https://crbug.com/663131) |
+GATT Server Connect       | ✓         | ✓       | ✓   | ✓     | ✓       |
 GATT Server Disconnect    | ✓         | ✓       | ✓   | ✓     |         |
```
Comment (cco3) [inline implementation-status.md]: Disconnect "works".  We just mark it disconnected, whether or not Windows does anything.  The same is the case for Connect.  We just mark it connected and expect Windows to establish a connection as needed.

**Comment by cco3** [comment]
Thanks @beaufortfrancois !

File: implementation-status.md (hunk 1 above)
Comment (beaufortfrancois) [reply implementation-status.md]: Noted.

**Comment by beaufortfrancois** [comment]
@cco3 will know which exact version of Chrome added Windows support.

**Comment by cco3** [comment]
These features were added in 65.

**Pull request merged**